scripts/generator/graphql_generator/csharp/Editor/ShopifyBuildPipeline.cs.erb

```diff
@@ -0,0 +1,61 @@
+namespace <%= namespace %>.SDK {
```

Comment:
When you talk about the sample app is that what @richardmonette is building? If so maybe this script should be distributed in that app. The same app won't live in this same repo.